tm1637.py

```diff
@@ -199,6 +199,34 @@ def temperature(self, num):
             self.write(self.encode_string(string))
         self.write([_SEGMENTS[38], _SEGMENTS[12]], 2)  # degrees C
 
+    def dectemperature(self, num):
+        if num < -9.9: # limit to single digit negatives
+            self.write([0, 0, 0, 0])
+            self.show('lo')  # low
+        elif num > 99.9:
+            self.write([0, 0, 0, 0])
+            self.show('hi')  # high
+        else:
+            intval = int(num/1)
```

Comment:
Here you can use `abs(int(num/1))` and remove the "abs" from the code below

Reply:
will do